content/learn/migration-guides/0.12-to-0.13.md

```diff
@@ -884,6 +884,14 @@ The `TextureAtlas` asset that previously contained both the atlas layout and ima
 
 `SpriteSheetBundle` now uses a `Sprite` instead of a `TextureAtlasSprite` component and a `TextureAtlas` component instead of a `Handle<TextureAtlaslayout>`.
 
+`DynamicTextureAtlasBuilder::add_texture` takes an addition `&Handle<Image>` parameter.
+
+`TextureAtlasLayout::from_grid` no longer takes a `Handle<Image>` parameter.
+
+`TextureAtlasBuilder::finish` now returns a `Result<(TextureAtlasLayout, Handle<Image>), _>`.
+
+`UiAtlasImage` was removed. The atlas bundle is now identical to `ImageBundle` with an additional `TextureAtlas`.
```

Comment:
```suggestion
`UiAtlasImage` was removed. Use `ImageBundle` and add an additional `TextureAtlas` component.
```

Reply:
This is confusing. https://docs.rs/bevy/latest/bevy/prelude/struct.AtlasImageBundle.html does exist; why shouldn't this be used like the changelog suggests?

Reply:
Oops, this is nonsense from me.

Reply:
I think that it was `UiTextureAtlasImage` that was removed though.